components/asic/include/pll.h

```diff
@@ -0,0 +1,13 @@
+#ifndef PLL_H_
```

Comment:
probably should have a full path for the include guard. ASIC_INCLUDE_PLL_H

Reply:
We don't do that for any of the others. Might be a good idea, but then it should be system wide in a separate PR.